ui.frontend/src/utils.js

```diff
@@ -309,14 +327,15 @@ class Utils {
             } else {
                 const _formJson = await HTTPAPILayer.getFormDefinition(_path, _pageLang);
                 console.debug("fetched model json", _formJson);
-                await this.registerCustomFunctions(_formJson.id);
```

Comment:
We could improve the existing function implementation to incorporate the following logic,
Check if fd:customFunctionsUrl exists, if yes, use it, else fallback to the existing logic

let's not create a new method

Reply:
The intention to create a new function was not to change the signature of the existing one. I see the existing function as exposed and documented.

Reply:
@barshat7 You were right, this function is a public API, https://opensource.adobe.com/aem-core-forms-components/module-FormView-HTTPAPILayer.html

Can we deprecate this API and create a new API with proper JS doc annotation in each API